readme.md

```diff
@@ -5,8 +5,15 @@
 [Test Coverage Report](https://eeca-nz.github.io/home-efficiency-calculator/htmlcov)
 [API documentation](http://home-efficiency-calculator.australiaeast.azurecontainer.io:8000/docs)
 
-This repository contains the source code for the home efficiency calculator, a FastAPI application designed to provide insights into household energy costs and CO2 emissions.
+This repository contains the source code for the [Home efficiency calculator](http://home-efficiency-calculator.australiaeast.azurecontainer.io:8000/docs), a FastAPI application designed to provide insights into household energy costs and CO2 emissions.
 
+It is used to serve the numbers behind EECA's public-facing [Home energy savings calculator](https://www.genless.govt.nz/for-everyone/at-home/energy-saving-appliances/home-energy-savings-calculator/) web app.
+
+We would like to thank the team behind the related [Powerswitch](https://www.powerswitch.org.nz/questionnaire) tool for their generosity in sharing a national tariff dataset, which has allowed us to build regional energy pricing into our model.
```

Comment:
Suggest removing "related" just in case this creates the impression that there is a shared codebase?

Reply:
Yeah agreed.